modules/reference/pages/cluster-properties.adoc

```diff
@@ -1638,6 +1638,24 @@ Target replication factor for internal topics.
 
 ---
 
+=== minimum_topic_replication
+
+Minimum permitted replication factor value for newly-created topics. Redpanda returns an `INVALID_REPLICATION_FACTOR` error on any attempt to create a topic with a replication factor less than this property.
+
+When changing the `minimum_topic_replication` parameter, any previously existing topics having a replication factor less than the new value will continue to work as normal. Redpanda will, however, issue a warning level log message at start up indicating the existence of topics with a replication factor less than the cluster's `minimum_topic_replication`. That error message will read, for example, `Topic X has a replication factor less than specified minimum: 1 < 3`.
```

Comment:
The point here was to make sure users know setting this won't update any other existing topics. If you have a topic with a replication factor of 1 and you set the minimum to 3, the previously existing topic will not be updated and it will continue functioning with a replication factor of 1. You'll just get error messages indicating the existence of a topic that breaks the configured lower bound.

Reply:
Yup. So I would change the passage that says `continue to work as normal` . Would include something along the lines of "modifying this parameter don't change the replication factor for existing topics"